source/renderer/app/components/widgets/forms/AssetsDropdown.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import React, { Component } from 'react';
+import { omit, filter, escapeRegExp } from 'lodash';
+import ItemsDropdown from './ItemsDropdown';
+import { formattedTokenWalletAmount } from '../../../utils/formatters';
+import type { WalletSummaryAsset } from '../../../api/assets/types';
+import AssetToken from '../../assets/AssetToken';
+import styles from './AssetsDropdown.scss';
+
+/**
+ *
+ * This component extends the ItemDropdownProps component
+ * which is based on React Polymorph's Select
+ * Any prop from it can be used
+ * Reference:
+ * https://github.com/input-output-hk/react-polymorph/blob/develop/source/components/Select.js
+ *
+ */
+type Props = {
+  assets?: Array<$Shape<WalletSummaryAsset>>,
+};
+
+export const onSearchAssetsDropdown = (
```

Comment:
I'd keep it as it is, because the prop name in the RP's `Select` component is `onSearch`, so `onSearchAssetsDropdown` is more similar. The params position are also derived from there, and the `searchValue` is the first one because in some cases you may simply ignore the array in second one, when you have already the list in a better format.

Reply:
Ok